_includes/releases/v22.2/v22.2.8.md

```diff
@@ -0,0 +1,88 @@
+## v22.2.8
+
+Release Date: April 17, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-8-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- [Backup schedules](../v22.2/manage-a-backup-schedule.html) created or altered to have the option `on_previous_running` will now have the full backup schedule created with the user specified option, but will override the incremental backup schedule to always default to `on_previous_running = wait`. This prevents duplicate incremental jobs from racing against each other, and ensures correctness of the backup chains created by the incremental schedule. [#98860][#98860]
+- [AVRO](../v22.2/changefeed-messages.html#avro) schema registry URI now allow additional `timeout=T` query parameter to change the default timeout for contacting schema registry. [#99505][#99505]
+
+<h3 id="v22-2-8-sql-language-changes">SQL language changes</h3>
+
+- Fixed the helper message on the [UPDATE](../v22.2/update.html) statement to correctly position the optional `FROM` clause. [#99299][#99299]
+- Added a new `prepared_statements_cache_size` [cluster setting](../v22.2/cluster-settings.html) which, when set to a non-zero number of bytes, causes the least recently-used prepared statements to be automatically deallocated when prepared statement memory usage goes above the cache size. This setting can be used to avoid prepared statement leaks from long-lived connections which never `DEALLOCATE` prepared statements. [#99259][#99259]
+- Added two new [cluster settings](../v22.2/cluster-settings.html) that enable users to change the number of histogram samples and buckets collected when building histograms as part of table statistics collection: `sql.stats.histogram_samples.count` and `sql.stats.histogram_buckets.count`. While the default values should work for most cases, it may be beneficial to increase the number of samples and buckets for very large tables to avoid creating a histogram that misses important values. [#100544][#100544]
```

Comment:
```suggestion
- Added two new [cluster settings](../v22.2/cluster-settings.html) that enable users to change the number of histogram samples and buckets collected when building histograms as part of table statistics collection: `sql.stats.histogram_samples.count` and `sql.stats.histogram_buckets.count`. While the default values should work for most cases, it may be beneficial to increase the number of samples and buckets for very large tables to avoid creating a histogram that misses important values. [#100544][#100544]
```
I'm not sure the While...values bit belongs in a release note. WDYT?

Reply:
Removed.